dev-support/determine-flaky-tests-hadoop.py

```diff
@@ -35,22 +35,8 @@
 # at the failed test for the specific run is necessary.
 #
 import sys
-import platform
-sysversion = sys.hexversion
-onward30 = False
-if sysversion < 0x020600F0:
-  sys.exit("Minimum supported python version is 2.6, the current version is " +
-      "Python" + platform.python_version())
-
-if sysversion == 0x030000F0:
-  sys.exit("There is a known bug with Python" + platform.python_version() +
-      ", please try a different version");
-
-if sysversion < 0x03000000:
-  import urllib2
-else:
-  onward30 = True
-  import urllib.request
+
```

Comment:
Is the `determine-flaky-tests-hadoop.py` is being used? I thought it does not work since the developers are manually filing the flaky tests.

Reply:
I think it is not used. Now the developers can check the age from Jenkins Web UI (https://ci-hadoop.apache.org/job/hadoop-qbt-trunk-java8-linux-x86_64/418/testReport/) instead of running the script.

![image](https://user-images.githubusercontent.com/3403122/108005235-83b52400-703b-11eb-97f9-cd45de8d4947.png)

Reply:
Removed the file.